### ISSUE TITLE
Progress towards v1.0

### ISSUE THREAD

**Issue by timbunce** [manyminds/api2go#284]
v1.0-RC2 was released this on Aug 16, 2016. There have been 31 commits to master since then.

Making a v1.0-RC3 release from master would be a nice way to add some visible momentum to the project.

There are currently just three issues open for the [v1.0 milestone](https://github.com/manyminds/api2go/milestone/2). Are they all blockers?

**Comment by wwwdata** [comment]
Hi @timbunce do you actively use api2go and want to help us out? This project is currently standing still because I and also @sharpner don't use api2go for current projects.

**Comment by timbunce** [comment]
I'm working on a proof-of-concept for using it. Unsure about going forward yet, though it's looking hopeful.

If we go ahead with api2go then I would be interested in helping out. I have experience implementing [something similar](https://metacpan.org/pod/WebAPI::DBIC) in Perl. I'm still very new Go though.

**Comment by xomaczar** [comment]
@wwwdata I thought you guys were using ember w/ JSONAPI and api2go? Is that not the case anymore?

**Comment by sharpner** [comment]
yes we stopped working on that project. But it could be that api2go is still in use there :)

**Comment by philippecarle** [comment]
As you've seen, I'm up to be involved in the project and will most likely rely on it for a quite big personal work on a large API. I would be interested in giving a hand to make it fully functional and production ready as from what I've tried by playing with it, it's exactly what I need. 
I'm using glide for vendoring and can't find a way to use the master branch so can't get benefits from updates (and even from my own PR 😩), so tagging would be a great benefit. Having a milestone to manage issues and features for the 1.0.0 release would be great !

**Comment by wwwdata** [comment]
We have a milestone for 1.0. Do  you think we should put anything else to the 1.0 milestone? Let's make it happen and implement all the tickets and then tag 1.0

**Comment by mirague** [comment]
Great initiative to keep the momentum going guys!

**Comment by xomaczar** [comment]
This is the best golang json:api library out there, would love to help finish v1
